test/unit/test.visual_recognition.v3.js

```diff
@@ -84,7 +84,7 @@ describe('visual_recognition', function() {
     nock.cleanAll();
   });
 
-  const visual_recognition = watson.visual_recognition(service);
+  const visual_recognition = watson.watson_vision_combined(service);
```

Comment:
why?

Reply:
According to swagger: `"x-vcap-service-name": "watson_vision_combined"`